frontend/src/component/application/ApplicationChart.tsx

```diff
@@ -0,0 +1,227 @@
+    fontSize: theme.fontSizes.smallerBody,
+    marginTop: theme.spacing(2),
+}));
+
+const StyledCell = styled('td')(({ theme }) => ({
+    verticalAlign: 'top',
+    paddingLeft: 0,
+    paddingRight: theme.spacing(1),
+}));
+
+const StyleApplicationContainer = styled(Box)(({ theme }) => ({
+    marginBottom: theme.spacing(18),
+    display: 'flex',
+    justifyContent: 'center',
+}));
+
+const StyledApplicationBox = styled(Box)<{
+    mode: 'success' | 'warning';
+}>(({ theme, mode }) => ({
+    borderRadius: theme.shape.borderRadiusMedium,
+    border: '1px solid',
+    borderColor: theme.palette[mode].border,
+    backgroundColor: theme.palette[mode].light,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: theme.spacing(1.5, 3, 2, 3),
+}));
+
+const StyledEnvironmentBox = styled(Box)<{
+    mode: 'success' | 'warning';
+}>(({ theme, mode }) => ({
+    borderRadius: theme.shape.borderRadiusMedium,
+    border: '1px solid',
+    borderColor:
+        theme.palette[mode === 'success' ? 'secondary' : 'warning'].border,
+    backgroundColor:
+        theme.palette[mode === 'success' ? 'secondary' : 'warning'].light,
+    display: 'inline-block',
+    padding: theme.spacing(1.5, 1.5, 1.5, 1.5),
+}));
+
+const StyledDivider = styled(Divider)(({ theme }) => ({
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    width: '100%',
+}));
+
+const StyledEnvironmentsContainer = styled(Box)({
+    display: 'flex',
+    justifyContent: 'start',
+    gap: '20px',
+});
+
+const EnvironmentHeader = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallerBody,
+    fontWeight: theme.fontWeight.bold,
+}));
+
+const StyledStatus = styled(Typography)<{
+    mode: 'success' | 'warning';
+}>(({ theme, mode }) => ({
+    gap: theme.spacing(1),
+    fontSize: theme.fontSizes.smallBody,
+    color: theme.palette[mode].dark,
+    display: 'flex',
+    alignItems: 'center',
+}));
+
+const useElementWidth = () => {
+    const elementRef = useRef<HTMLDivElement>(null);
+    const [width, setWidth] = useState('100%');
+
+    useLayoutEffect(() => {
+        setWidth(`${elementRef.current?.scrollWidth}px`);
+    }, [elementRef, setWidth]);
+
+    return {
+        elementRef,
+        width,
+    };
+};
+const SuccessStatus = () => (
+    <StyledStatus mode='success'>
+        <WarningAmberRounded
+            sx={(theme) => ({
+                color: theme.palette.success.main,
+            })}
+        />{' '}
+        Everything looks good!
+    </StyledStatus>
+);
+
+const WarningStatus: FC = ({ children }) => (
+    <StyledStatus mode='warning'>
+        <WarningAmberRounded
+            sx={(theme) => ({
+                color: theme.palette.warning.main,
+            })}
+        />{' '}
+        {children}
+    </StyledStatus>
+);
+
+interface IApplicationChartProps {
+    data: ApplicationOverviewSchema;
+}
+
+export const ApplicationChart = ({ data }: IApplicationChartProps) => {
```

Comment:
Moved this out out, because the main application overview was growing too large.